### DISCUSSION TITLE
Animation

### DISCUSSION THREAD

**Discussion by Darkzpt** [naverz/zepeto-studio-kor#219] [5. Q&A]
여기 3D 항목에 대한 zepeto 애니메이션을 가르쳐 줄 수 있는 사람이 있습니까? 내 항목을 만들고 싶지만 Unity에서 막혔습니다.

**Comment by x4cf** [comment]
https://github.com/naverz/zepeto-studio-kor/discussions/154

**Reply by Darkzpt** [reply]
어떤 이유로 나는 그것을 이해하지 못합니다.

**Reply by Darkzpt** [reply]
저는 마야와 유니티를 사용하고 있습니다. 조작되고 애니메이션이 있는 모델이 있지만 Unity로 이동하면 애니메이터 컨트롤러가 있어도 작동하지 않습니다.
